src/lib/utils/liquidityMining/LiquidityMiningV2.json

```diff
@@ -0,0 +1,27 @@
+{
+    "week_51": {
+      "start_timestamp": 1621220400,
```

Comment:
Although I initially suggested the `start_timestamp` and `end_timestamp` keys, I'd like to propose we remove them. They are not used in the frontend and I don't expect we'll have a need for them in the off-chain scripts either.